project/SensibleSettings.scala

```diff
@@ -13,7 +13,8 @@ object Sensible {
 
   lazy val settings = Seq(
     ivyLoggingLevel := UpdateLogging.Quiet,
-    conflictManager := ConflictManager.strict,
+    // FIXME: how to be less strict for typesafe config? slick and akka pull in different versions
+    //conflictManager := ConflictManager.strict,
```

Comment:
How do you deal with conflicts like this when it's likely or known that the versions are actually compatible?